src/ds5/ds5-color.cpp

```diff
@@ -6,21 +6,38 @@
 #include <vector>
 #include <iterator>
 #include <cstddef>
+#include <functional>
 
 #include "device.h"
 #include "context.h"
 #include "image.h"
 #include "metadata-parser.h"
+#include "global_timestamp_reader.h"
+#include "environment.h"
 
 #include "ds5-color.h"
 #include "ds5-private.h"
 #include "ds5-options.h"
 #include "ds5-timestamp.h"
-#include "global_timestamp_reader.h"
-#include "environment.h"
+
+#include "proc/color-formats-converter.h"
+#include "proc/identity-processing-block.h"
```

Comment:
This feels like should be in the sensor include (the identity, not color-formats)